master/buildbot/test/unit/test_changes_gitpoller.py

```diff
@@ -285,6 +285,19 @@ def cb(_):
                 'master': '4423cdbcbb89c14e50dd5f4152415afd686c5241'
             })
 
+    def test_poll_GitError(self):
+        # Raised when git exits with status code 128. See issue 2468
+        self.expectCommands(
+            gpo.Expect('git', 'init', '--bare', 'gitpoller-work')
+            .exit(128),
+        )
+
+        d = self.assertFailure(self.poller._dovccmd('init', ['--bare',
```

Comment:
So should I remove `inlineCallbacks`? (And, am I using `inlineCallbacks` correctly?)